dxm/src/main/java/io/pcp/parfait/dxm/PcpOffset.java

```diff
@@ -5,4 +5,5 @@
 
 interface PcpOffset {
```

Comment:
I'll agree that `PcpOffset` isn't the best abstraction anymore. It is basically used as a way to setup the implementer of the interface for writing _later_. It felt like the offset was also related to the length of the data structure but perhaps it should implement another interface? I think specifically mentioning bytes in either a method or an interface is needed. I'm not a huge fan of `length` as length can be different depending on the size of numbers, character encoding etc...